backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/BinaryInterface.kt

```diff
@@ -85,7 +85,16 @@ internal val IrClass.writableTypeInfoSymbolName: String
         return "ktypew:" + this.fqNameForIrSerialization.toString()
     }
 
-internal val IrClass.objectInstanceGetterSymbolName: String
+internal val IrClass.globalObjectInstanceSymbolName: String
```

Comment:
It's not an "instance" symbol name, but a name for "instance storage". Same below.